scripts/build.sh

```diff
@@ -15,9 +15,30 @@ cd lib/renderer
 mewn
 cd ../..
 
-echo "**** Installing Wails locally ****"
+echo "**** Checking if Wails passes unit tests ****"
```

Comment:
Nice

Reply:
yeah - if only there were more tests :)

Reply:
It's a pretty hard setup to test to be fair.